package.json

```diff
@@ -7,9 +7,12 @@
     "ios": "expo start --ios",
     "start": "expo start",
     "test": "jest --verbose --coverage",
-    "web": "expo start --web"
+    "web": "expo start --web",
+    "lint": "tsc --noEmit && eslint --ext .ts,.tsx ./",
+    "lint-fix": "tsc --noEmit && eslint --fix --ext .ts,.tsx ./"
   },
   "dependencies": {
+    "@expo/vector-icons": "^10.0.6",
```

Comment:
I believe that this is not necessary. this package in included in `expo`
https://docs.expo.io/versions/latest/guides/icons/

Reply:
You're right it passes locally without the package dependency, but it fails on Travis which is why we need it.